### ISSUE TITLE
How do you get the min- and max-value for clipping? 

### ISSUE THREAD

**Issue by wuzhongwulidong** [waychin-weiqin/ITSA#4]
Great work! 
Would you provide some information about how you get the magic numbers of the min- and max-values for clipping as  [-2.1179, -2.0357, -1.8044] and [2.2489, 2.4286, 2.6400]?
Thanks!

    def clip(self, img, img_min=None, img_max=None):
        if img_min is None:
            img_min = torch.tensor([-2.1179, -2.0357, -1.8044]).view(1,3,1,1).cuda()

        if img_max is None:
            img_max = torch.tensor([2.2489, 2.4286, 2.6400]).view(3,1,1).cuda()

        img = torch.clip(img, min=img_min, max=img_max)

        return img

**Comment by waychin-weiqin** [comment]
Hi @wuzhongwulidong , thank you for showing interest in our work. These values are the max and min pixel values normalized using the mean and std of ImageNet. 
![image](https://user-images.githubusercontent.com/33308266/197428450-43592732-4263-46e1-a9f1-9d620bcb01fa.png)

**Comment by wuzhongwulidong** [comment]
Thank you very much. Got it.
